.buildkite/scripts/create-github-release.sh

```diff
@@ -34,7 +40,7 @@ if [ -n "${BUILDKITE_TAG}" ] ; then
     --draft \
     --title "${BUILDKITE_TAG}" \
     --repo elastic/synthetics-recorder \
-    "${DIST_LOCATION}/*.*"
+    ./$DIST_LOCATION/*.*
```

Comment:
NIT: without looking at the details.

`DIST_LOCATION` could be an absolute path. which would make this path wrong.

Reply:
it's not, see [L9](https://github.com/elastic/synthetics-recorder/pull/539/files#diff-275b8b0b9f438320f9038499a9ed4a17b7c185a9145ccbebcdf4cde0e0d127ffR9)